worldedit-bukkit/src/main/java/com/sk89q/worldedit/bukkit/BukkitWorld.java

```diff
@@ -123,31 +123,6 @@ public BukkitWorld(World world) {
         return list;
     }
 
-    @Nullable
-    @Override
-    public com.sk89q.worldedit.entity.Entity createEntity(com.sk89q.worldedit.util.Location location, BaseEntity entity) {
-        BukkitImplAdapter adapter = WorldEditPlugin.getInstance().getBukkitImplAdapter();
-        if (adapter != null) {
-            try {
-                Entity createdEntity = adapter.createEntity(BukkitAdapter.adapt(getWorld(), location), entity);
-                if (createdEntity != null) {
-                    return new BukkitEntity(createdEntity);
-                } else {
-                    return null;
-                }
-            } catch (Exception e) {
-                logger.warn("Corrupt entity found when creating: " + entity.getType().getId());
-                if (entity.getNbtData() != null) {
-                    logger.warn(entity.getNbtData().toString());
-                }
-                e.printStackTrace();
-                return null;
-            }
-        } else {
-            return null;
-        }
-    }
-
```

Comment:
I'm not sure this needs to be removed with the interest of upstream compatibility

Reply:
i think there were issues with async entity add (99% sure)

Reply:
it was "moved" to the IChunkExtent

Reply:
yeah it was that -> stacktrace https://waste.cutiepie.at/xd1mm8uoixmoxlsx

Reply:
Please insert a comment in this file explaining why it was removed for future reference.

Reply:
Done in https://github.com/IntellectualSites/FastAsyncWorldEdit/pull/518/commits/7f0fd182bb4c10e02a0db8465c519e5be7d4ef2c